packages/webview_flutter/webview_flutter_wkwebview/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## NEXT
+
+* Starts the Dart side of the platform specific implemention for iOS. Created
+  `WebViewCupertinoWidget`and added support for `WebSettings.allowsInlineMediaPlayback` and
+  `AutoMediaPlaybackPolicy`. This currently has no effect on the working iOS implementation.
```

Comment:
I think we shouldn't put this in the CHANGELOG. If we end up with bugfix releases to this package interspersed with the work I think it would make for a very confusing changelog. Let's do the PR description exemption for all of these until it's actually turned on.

This would all be good detail to add to the PR description though.